pco/src/standalone/decompressor.rs

```diff
@@ -195,7 +195,7 @@ impl<T: NumberLike, R: BetterBufRead> ChunkDecompressor<T, R> {
   }
 
   // a helper for some internal things
-  pub(crate) fn decompress_remaining_extend(&mut self, dst: &mut Vec<T>) -> PcoResult<()> {
+  pub fn decompress_remaining_extend(&mut self, dst: &mut Vec<T>) -> PcoResult<()> {
```

Comment:
Had to make this less restrictive in order to use it in `auto_decompress` from the `pco_python` crate.

Reply:
I'd rather not add this one to the public API as it is. I would suggest we just copy+paste what we need from this function for now (using `chunk_decompressor.n()` instead of `remaining`). If you feel strongly, we could add this function to the public API, but we'd need to add documentation and improve the bad name I gave it.